### PR TITLE
PHP 7.1+ only

### PR DESCRIPTION
**Pull request by alexeyshockov**
- [x] Run `composer normalize` to structure `composer.json`
- [x] Bump PHP version
- [x] Add branch a alias (`0.2.x-dev`)
- [x] Update (dev) deps
- [x] Update tests for PHPUnit 7

### CODE REVIEW

**Comment by alexeyshockov** [comment]
I also added a branch alias for the next version, because changing the PHP version requirements is definitely a BC break

**Comment by jcchavezs** [comment]
I think I will create a 2.0 branch in zipkin-php so we don't use just `dev-master`.

File: tests/Unit/ScopeManagerTest.php [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 namespace ZipkinOpenTracing\Tests\Unit;
 
-use PHPUnit_Framework_TestCase;
-use ZipkinOpenTracing\ScopeManager;
 use OpenTracing\Span;
+use PHPUnit\Framework\TestCase;
+use ZipkinOpenTracing\ScopeManager;
 
-final class ScopeManagerTest extends PHPUnit_Framework_TestCase
+final class ScopeManagerTest extends TestCase
```
Comment (jcchavezs) [inline tests/Unit/ScopeManagerTest.php:9]: :+1:

**Comment by alexeyshockov** [comment]
Changed to `v2.x-dev`. Let me know if it's correct or not.

**Review by jcchavezs — APPROVED** [review]

**Comment by jcchavezs** [comment]
Merged, please let me know if you manage to give it a try. Also worth to check `zipkin-php` and see if there are missing `php 7.1` features.

**Pull request merged**